src/box/lua/tuple_format.c

```diff
@@ -57,9 +57,7 @@ lbox_tuple_format_gc(struct lua_State *L)
 static int
 lbox_tuple_format_new(struct lua_State *L)
 {
-	int top = lua_gettop(L);
-	(void)top;
-	assert((1 <= top && 2 >= top) && lua_istable(L, 1));
+	luaT_checktype(L, 1, LUA_TTABLE);
```

Comment:
Don't you plan to fix other assertion failures here?
```
box.internal.tuple_format.new({type = 'test'})
Assertion failed: (mp_typeof(**data) == MP_ARRAY), function field_def_array_decode, file field_def.c, line 584.
```

Reply:
Thanks, missed it.